problems/4.arrays.js

```diff
@@ -1,31 +1,60 @@
 /** Averages the first and last value of an array.  Return null if the array has fewer than 2 elements.
-* @param {number[]} arr - The input array
-* @returns {number} - The average of the first and last values
-*/
-function firstAndLastAverage() {
-
-}
+ * @param {number[]} arr - The input array
+ * @returns {number} - The average of the first and last values
+ */
+//function firstAndLastAverage() {}
+const firstAndLastAverage = (arr) => {
```

Comment:
top line should be your checki. 
if(arr.length < 2) return null. 
Then you can just have the rest of the code. It will make it more readable.